<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>350</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Edit Note</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>../icons/edit_note.png</normalon>
   </iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QDateEdit" name="date_date">
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::UpDownArrows</enum>
     </property>
     <property name="currentSection">
      <enum>QDateTimeEdit::MonthSection</enum>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="currentSectionIndex">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="submit_button">
     <property name="text">
      <string>Submit Report Edit</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QTextEdit" name="edit_tedit">
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="autoFormatting">
      <set>QTextEdit::AutoNone</set>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="tabStopWidth">
      <number>20</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edit_tedit</tabstop>
  <tabstop>date_date</tabstop>
  <tabstop>submit_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
